src/Integrations/Integrations/WordPress/V4/WordPressIntegrationLoader.php

```diff
@@ -35,182 +37,311 @@ public function load(WordPressIntegration $integration)
 
         // Core
         \DDTrace\trace_method('WP', 'main', function (SpanData $span) use ($service) {
-            $span->name = $span->resource = 'WP.main';
-            $span->type = Type::WEB_SERVLET;
-            $span->service = $service;
+            WordPressIntegrationLoader::setCommonValues(
+                $span,
+                'WP.main',
+                Type::WEB_SERVLET,
+                $service,
+                'WP.main'
+            );
```

Comment:
I don't like these changes, I think it was more readable and explicit before.
I'd rather recommend to just copypaste the Tag::COMPONENT setting here.

Reply:
reverted these back to before and just copy/pasted component for each function